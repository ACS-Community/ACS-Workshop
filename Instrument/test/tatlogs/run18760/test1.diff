6d5
< 1 - Ran 1 test in 0.176s

