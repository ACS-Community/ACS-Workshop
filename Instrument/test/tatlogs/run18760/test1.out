1 - .
1 - ----------------------------------------------------------------------
1 - 
1 - OK
1 - CAMERA ON
